sdk/storage/azure_storage_blob/tests/blob_container_client.rs

```diff
@@ -32,3 +34,68 @@ async fn test_create_container(ctx: TestContext) -> Result<(), Box<dyn Error>> {
     container_client.delete_container(None).await?;
     Ok(())
 }
+
+#[recorded::test]
+async fn test_get_container_properties(ctx: TestContext) -> Result<(), Box<dyn Error>> {
+    // Recording Setup
+    let recording = ctx.recording();
+    let (options, endpoint) = recorded_test_setup(recording, BlobClientOptions::default()).await;
+    let container_name = recording
+        .random_string::<17>(Some("container"))
+        .to_ascii_lowercase();
+
+    // Act
+    let container_client = ContainerClient::new(
+        &endpoint,
+        container_name,
+        recording.credential(),
+        Some(options),
+    )?;
+    container_client.create_container(None).await?;
+    let response = container_client
+        .get_container_properties(Some(BlobContainerClientGetPropertiesOptions::default()))
+        .await;
+
+    // Assert
+    assert!(response.is_ok());
```

Comment:
I think it would be fine to leave this out and move the `?` below to line 55. If the request failed, it test will fail because of the `?` which is fine.